src/doc/book/src/reference/specifying-dependencies.md

```diff
@@ -308,7 +308,7 @@ resolve to 1.0.1 inside the repository instead of trying to download a version
 from crates.io. Once 1.0.1 is published on crates.io the `[patch]` section can
 be deleted.
 
-It's also worth nothing that `[patch]` applies *transitively*. Let's say you use
+It's also worth nothing that `[patch]` applies at the *top level*. Let's say you use
```

Comment:
This I believe isn't correct, the point of `[patch]` is to apply transitively to the entire crate graph.

Reply:
So the second declaration is the incorrect one? (just a few lines below)

Reply:
Sort of. I think the one below just needs to be clarified. The `[patch]` section can only be defined at the top-level but its effects are transitive throughout the crate graph.